### PR TITLE
Experiment webhook for metricsCollector

### PR DESCRIPTION
**Pull request by hougangliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/749)

### CODE REVIEW

File: manifests/v1alpha3/katib-controller/katib-config.yaml [hunk 1]
```diff
@@ -6,13 +6,13 @@ metadata:
...
-      "stdout": {
+      "StdOut": {
         "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/stdout-metricscollector"
       },
-      "file": {
+      "File": {
         "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/file-metricscollector"
       },
-      "tensorflow-event": {
+      "TensorflowEvent": {
```
Comment (gaocegege) [inline manifests/v1alpha3/katib-controller/katib-config.yaml]:
```suggestion
      "TensorFlowEvent": {
```

Comment (hougangliu) [reply manifests/v1alpha3/katib-controller/katib-config.yaml]: Done！thanks

File: pkg/webhook/v1alpha3/experiment/validator/validator.go [hunk 2]
```diff
@@ -208,5 +211,41 @@ func (g *DefaultValidator) validateMetricsCollector(inst *experimentsv1alpha3.Ex
 	if mcjob.GetNamespace() != namespace || mcjob.GetName() != trialName {
 		return fmt.Errorf("Invalid metricsCollector template.")
 	}
+
+	// Above part of this method will be dropped
+	mcSpec := inst.Spec.MetricsCollectorSpec
+	mcKind := mcSpec.Collector.Kind
+	// TODO(hougangliu):
```
Comment (gaocegege) [inline pkg/webhook/v1alpha3/experiment/validator/validator.go]: Should we remove the TODO here?

Comment (hougangliu) [reply pkg/webhook/v1alpha3/experiment/validator/validator.go]: yes, we need it since we'd better enhance here in future

**Pull request merged**